### ISSUE TITLE
We should print the ssh-to-builder instructions a bit earlier

### ISSUE THREAD

**Issue by praiskup** [fedora-copr/copr#3180]
```
[2024-03-07 13:49:08,349][  INFO][PID:3914053] The expiration time was set to 1709822947.6179445
[2024-03-07 13:49:08,350][  INFO][PID:3914053] Starting remote build: copr-rpmbuild --verbose --drop-resultdir --task-url https://copr.devel.redhat.com/backend/get-build-task/110654-rhel-7-x86_64 --chroot rhel-7-x86_64 --detached
[2024-03-07 13:49:09,095][  INFO][PID:3914053] Downloading the builder-live.log file, attempt 1
[2024-03-07 13:49:09,097][  INFO][PID:3914053] Popen command started: ssh -F /home/copr/.ssh/config mockbuilder@<ip> copr-rpmbuild-log
[2024-03-07 13:49:14,098][  INFO][PID:3914053] Status: False
[2024-03-07 13:51:09,824][  INFO][PID:3914053] 

Keeping builder alive for user SSH
[2024-03-07 13:51:09,825][  INFO][PID:3914053] The owner of this build can connect using:
[2024-03-07 13:51:09,825][  INFO][PID:3914053] ssh root@<IP>
```

It would be nice to have the report before we start the `copr-rpmbuild-log`
process which runs till the build finishes.
